src/main/java/duke/task/ToDos.java

```diff
@@ -0,0 +1,37 @@
+package duke.task;
+
+/**
+ * Encapsulates a task to be done.
+ */
+public class ToDos extends Task {
```

Comment:
You might want to rename the class from `ToDos` to `ToDo` since the class represents a `ToDo` task.

Plural form should be used on names representing a collection of objects.

Reply:
Thanks for the work mate! I've actually been wondering whether they should be plural due to  week 2 level 4's way of introducing the task.